sqrt_class.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
```

Comment:
Should we remove this file? It seems redundant with `_plot_sqrt_lasso.py`

Reply:
definitely !